packages/data/src/registry.js

```diff
@@ -171,5 +180,9 @@ export function createRegistry( storeConfigs = {} ) {
 		...storeConfigs,
 	} ).map( ( [ name, config ] ) => registry.registerStore( name, config ) );
 
+	if ( parent ) {
+		parent.subscribe( globalListener );
```

Comment:
It's nice how simply this works 👍 

It may prove to cause #13177 to be more difficult to implement, if we choose to pursue it again in the future (specifically, the change in signature to `subscribe( listener[, reducerKeys ] )`.